docker/Dockerfile.build.aarch64-linux

```diff
@@ -84,7 +84,7 @@ RUN /bin/bash -c 'DALI_DEPS_VERSION_SHA=${DALI_DEPS_VERSION_SHA:-$(cat /tmp/DALI
 # hack - install cross headers in the default python paths, so host python3-config would point to them
 RUN export PYVERS="3.6.9 3.7.8 3.8.5 3.9.0 3.10.0" && \
     for PYVER in ${PYVERS}; do \
-        cd /tmp && curl -L https://www.python.org/ftp/python/${PYVER}/Python-${PYVER}.tgz | tar -xzf - && \
+        cd /tmp && curl -L https://www.python.org/ftp/python/${PYVER}/Python-${PYVER}.tgz | tar -xzf - || exit 1 && \
```

Comment:
Shouldn't there be braces around `||`? Like
```bash
(curl -L https://www.python.org/ftp/python/${PYVER}/Python-${PYVER}.tgz | tar -xzf - || exit 1)
```
But that's only an impression, I'm not bash aficionado ;)

Reply:
Fixed

Reply:
It's actually the same effect:
```
> echo "A" && true || echo "C"
A

> echo "A" && false || echo "C"
A
C
```

Reply:
It has but it @szalpal variant may improve the readability